src/IntercomChannel.php

```diff
@@ -76,6 +77,12 @@ public function send($notifiable, Notification $notification): void
                 $message->toArray()
             );
         } catch (BadResponseException $exception) {
+            event(new NotificationFailed($notifiable, $notification, $this, [
```

Comment:
You don't have to do like that. This is unattended behavior, also event is not obvious dependency, there is EventDispatcher class exists for that
You must catch this Exception on your business layer

Reply:
Could you point me the proper way to catch that? In my testing, once the Notification has been given over to Laravel there's no way to catch errors it throws

Reply:
Try to read manual and internal Laravel code more accurately. 
Unfortunately, I can't help you with this now